aesara/tensor/random/basic.py

```diff
@@ -73,18 +73,18 @@ def rng_fn(cls, rng, loc, scale, size):
 
 
 class GammaRV(RandomVariable):
-    name = "halfnormal"
+    name = "gamma"
     ndim_supp = 0
     ndims_params = [0, 0]
     dtype = "floatX"
     _print_name = ("Gamma", "\\operatorname{Gamma}")
 
     def __call__(self, shape, rate, size=None, **kwargs):
-        return super().__call__(shape, 1.0 / rate, size=size, **kwargs)
+        return super().__call__(shape, rate, size=size, **kwargs)
```

Comment:
Why this breaking change?

Reply:
No good reason, just wanted to check if it broke things. I thought the same variable names rate / scale should be used between `__call__` and `rng_fn` and I changed it here. Will revert